packages/editor/src/client/i18n.ts

```diff
@@ -5,12 +5,27 @@ import LanguageDetector from 'i18next-browser-languagedetector'
 import en from './locales/en.json'
 import de from './locales/de.json'
 import fr from './locales/fr.json'
-import moment from 'moment'
+
+import {format as formatDate, isDate} from 'date-fns'
+import {de as deLocale, enGB as enLocale, fr as frLocale} from 'date-fns/locale'
+
+export interface DateFormatLocale {
+  lng: string
+  locale: Locale
+}
+const DateFormatLocales: DateFormatLocale[] = [
```

Comment:
I suggest you use a javascript map here instead of an array. It will be easier to get the locale on line 26.